base/special/exp.jl

```diff
@@ -35,28 +35,36 @@ const LN2 = 6.931471805599453094172321214581765680755001343602552541206800094933
 const LOG2_E = 1.442695040888963407359924681001892137426646
 
 # log(2) into upper and lower bits
-LN2U(::Type{Float64}) = 6.93147180369123816490e-1
 LN2U(::Type{Float32}) = 6.9313812256f-1
 
-LN2L(::Type{Float64}) = 1.90821492927058770002e-10
 LN2L(::Type{Float32}) = 9.0580006145f-6
 
+
+# -log(2)/1024 into upper and lower bits
+LN2o1024U(::Type{Float64}) = -6.769015435155716e-4
+LN2o1024L(::Type{Float64}) = -2.264694154146777e-20
+# 1024/log(2)
+LN2o1024INV(::Type{Float64}) = 1477.3197218702985
+
+# magic rounding constant: 1.5*2^52 Adding, then subtracting it from a float rounds it to an Int.
+MAGIC_ROUND_CONST(::Type{Float64}) = 6755399441055744.0
+
 # max and min arguments
-MAX_EXP(::Type{Float64}) = 7.09782712893383996732e2 # log 2^1023*(2-2^-52)
+MAX_EXP(::Type{Float64}) = 709.782712893383996732
 MAX_EXP(::Type{Float32}) = 88.72283905206835f0      # log 2^127 *(2-2^-23)
 
 # one less than the min exponent since we can sqeeze a bit more from the exp function
-MIN_EXP(::Type{Float64}) = -7.451332191019412076235e2 # log 2^-1075
+MIN_EXP(::Type{Float64}) = -745.1332191019412076235
```

Comment:
Why removed the comments?

Reply:
Mainly because this PR is in progress. I had taken out the entire lines and when I added them back, it was missing the comment.